messaging/src/main/java/org/axonframework/queryhandling/SimpleQueryHandlingComponent.java

```diff
@@ -44,10 +44,10 @@ public SimpleQueryHandlingComponent() {
     public MessageStream<QueryResponseMessage> handle(@Nonnull QueryMessage query,
```

Comment:
This component was a playground. Is it still relevant? :thinking:

Reply:
I will revamp it as part of #3488! :-)